foundations/html_css/flexbox/flexbox-intro.md

```diff
@@ -27,7 +27,9 @@ Flexbox is a way to arrange items into rows or columns. These items will flex (i
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-We'll get into exactly what's going on here soon enough. But for now, uncomment the two flex related lines in the above CSS and check out the result. All 3 divs should now be arranged horizontally. If you resize the results frame with the "1x", ".5x" and ".25x" buttons you'll also see that the divs will 'flex'. They will fill the available area and will each have equal width.
+We'll get into exactly what's going on here soon enough. But for now, uncomment the two flex related lines in the above CSS and check out the result. CSS comments prevent the browser from interpreting lines as code, and are wrapped between `/*` and `*/` tags. The lines can be 're-enabled' by removing the comment tags surrounding unused code. 
```

Comment:
Like the addition of mentioning comments within code! Wondering if this paragraph could be worded a bit differently for a better flow. What do you think of something along the lines of:

```suggestion
We'll get into exactly what's going on here soon enough. But for now, let's uncomment the two flex related CSS declarations in the above Codepen by removing the `/*` and `*/` tags surrounding them, then check out the result.

**A Note on Comments**: Comments prevent the browser from interpreting lines as code, and are wrapped between specific tags. CSS uses `/*`as an opening comment tag and `*/` as a closing comment tag, while HTML and JavaScript have their own syntax. Commented out lines of code can be 're-enabled' simply by removing the comment tags surrounding the code. 
```

Reply:
I had originally intended on adding a note similar to this, but I felt adding a note about comments closely preceding the note about the codepen created a slight distraction in the flow of the lesson. 

After looking over it again with the preview tool, the way you inserted the note seems a lot less distracting than the way I would have. I like it!

Reply:
Hey @takinabradley , just wanted to check if you were still looking to make this update?

Reply:
Oh, sorry! I didn't realize I had to push your suggested changes, and was waiting to hear back from you. It's done now!